### PR TITLE
Added MobileHeader Component UI

### PR DESCRIPTION
**Pull request by PrinceRajRoy**
Issue: #1324
Design: https://www.figma.com/file/CS01VVLR7ArQl0afYFkNj3/Web-App?node-id=4300%3A2250

### CODE REVIEW

**Review by goldyydev — CHANGES_REQUESTED** [review]
Just a minor issue but in the ProfileHeader, the Avatar is supposed to be larger by 10px than the icons.

**Comment by PrinceRajRoy** [comment]
@goldyydev, yup but the rest of the predefined `SingleUser` sizes are a lot bigger, should I add a new one, `md` ?

File: kibbeh/src/stories/MobileHeader/ProfileHeader.story.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  component: ProfileHeader,
+};
+
+const TheProfileHeader: Story<ProfileHeaderProps> = ({
+  avatar = src,
+  onAnnouncementsClick = () => null,
+  onMessagesClick = () => null,
+  onSearchClick = () => null,
+}) => (
+  <div style={{ width: 420 }}>
```
Comment (wes337) [inline kibbeh/src/stories/MobileHeader/ProfileHeader.story.tsx:22]: I think you don't need to set this width in the story container, there is a "small/large mobile" view setting in storybook.

**Comment by bansalarnav** [comment]
> 
> 
> @goldyydev, yup but the rest of the predefined `SingleUser` sizes are a lot bigger, should I add a new one, `md` ?

There is a `xxs` size in `SingleUser` which is 30px and matches your requirements

**Comment by goldyydev** [comment]
is there any progress on this?

**Comment by PrinceRajRoy** [comment]
> is there any progress on this?

I have used xs on the Profile image, xxs is bigger than that, is it supposed to be like this? xxs >xs?
I will push the fix, if that's correct.

snip

**Comment by goldyydev** [comment]
We really need it done, just make it work please.

**Comment by overlisted** [comment]
what are we waiting for

**Comment by goldyydev** [comment]
Come on....

**Pull request merged**